app.js

```diff
@@ -1,9 +1,26 @@
 require('dotenv').config();
-var express = require('express');
-var app = express();
+const express = require('express');
+const app = express();
+const dynamoose = require('dynamoose');
 
-// app.js
+// DynamoDB connection
+dynamoose.AWS.config.update({
+  accessKeyId: process.env.DYNAMO_ACCCESSKEYID,
+  secretAccessKey: process.env.DYNAMO_SECRETACCESSKEY,
+  region: process.env.AWS_REGION,
+  endpoint: process.env.DYNAMO_ENDPOINT
+});
+if (
+  process.env.DYNAMO_LOCAL &&
+  (process.env.DYNAMO_LOCAL === 'true' || process.env.DYNAMO_LOCAL === '1')
+) {
```

Comment:
What needs to be installed if we wanted to run it locally?

Reply:
Instructions here: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html

Our code assumes you are running local Dynamo on default port of 8000.